src/frontend/src/lib/components/transactions/TransactionModal.svelte

```diff
@@ -67,6 +67,8 @@
 
 		<slot name="transaction-status" />
 
+		<slot name="confirmations" />
```

Comment:
We have `transaction-status` but `confirmation`, should it be `transaction-confirmations` or should we rename the other or that's not really related?

Reply:
True, renamed it to `transaction-confirmations`.